packages/react-widgets/src/layers/EditableCartoGeoJsonLayer.js

```diff
@@ -0,0 +1,30 @@
+import { EditableGeoJsonLayer } from '@nebula.gl/layers';
+
+const EVENT_TYPES = ['anyclick', 'pointermove', 'panstart', 'panmove', 'panend', 'keyup'];
+
+export default class EditableCartoGeoJsonLayer extends EditableGeoJsonLayer {
+  _addEventHandlers() {
+    const eventManager = this._getEventManager();
+    const { eventHandler } = this.state._editableLayerState;
+
+    for (const eventType of EVENT_TYPES) {
+      eventManager.on(eventType, eventHandler, {
+        // give nebula a higher priority so that it can stop propagation to deck.gl's map panning handlers
+        priority: 100
```

Comment:
Is 100 the "top priority" or how does is work? I mean, is it an arbitrary number?.

Reply:
I really don't know XD. This code has been copied from the original nebula code
https://github.com/uber/nebula.gl/blob/master/modules/layers/src/layers/editable-layer.ts#L83

Reply:
XD, ok then add this link to code...

Reply:
Done